docs/contributing.md

```diff
@@ -133,24 +133,23 @@ recommended steps.
 One-time setup:
 
     rustup toolchain add nightly  # wanted for 'rustfmt'
-    rustup toolchain add 1.76     # also specified in Cargo.toml
-    cargo install cargo-insta
-    cargo install cargo-watch
-    cargo install cargo-nextest
+    cargo install cargo-insta cargo-watch cargo-nextest
 
 During development (adapt according to your preference):
 
     cargo watch --ignore '.jj/**' -s \
       'cargo clippy --workspace --all-targets \
-       && cargo +1.76 check --workspace --all-targets'
+       && cargo check --workspace --all-targets'
```

Comment:
comment (non-blocking): It occurs to me that the old instructions here were probably wrong or suboptimal since `cargo clippy` and `cargo check` were potentially running with different compiler versions, so they might not have reused the same build artifacts, or you could be getting too-old or too-new `clippy` diagnostics.